nautobot/project-static/css/base.css

```diff
@@ -42,7 +42,7 @@ body {
     transition-duration: var(--navbar-transition-duration);
     transition-timing-function: ease-in-out;
     transition-property: margin-left, width;
-    overflow: visible;
+    overflow: hidden;
```

Comment:
Maybe for now make overflow scroll, till the footer responsiveness is fixed 

```suggestion
    overflow-y: hidden;
    overflow-x: scroll;
```

Reply:
footer responsiveness was fixed in #5172

Reply:
so we should be ok to change to hidden for the footer.

Reply:
It does not look fixed

![Screenshot 2024-03-13 at 4 10 35 PM](https://github.com/nautobot/nautobot/assets/94907097/e0b95a08-618f-407a-a19b-c175128e3d61)

Reply:
Referring to the Help button?  If so, I can look into that.  The scrollbar does not fix that however.  It is an issue regardless.  

Having the scrollbar causes a worse experience in my opinion because it causes the screen to jump up and down when scroll the window from left to right.

Reply:
I just adjusted the css on the middle column of the footer to address your comment.  The help button no longer hangs over the right edge on medium width